test/controllers/users_controller_test.js

```diff
@@ -0,0 +1,38 @@
+const usersRoutes = require("../../index");
+const chai = require("chai"),
+    chaiHttp = require("chai-http");
+const should = chai.should();
+
+chai.use(chaiHttp);
+
+describe("Endpoints exists", () => {
+    describe("/GET users", () => {
+        it("the get users endpoint should exist", (done) => {
+            chai.request(usersRoutes).
+                get("/users").
+                end((err, res) => {
+                    should.not.exist(err);
+                    res.should.have.status(200);
+                    done();
+                });
+        });
+    });
+
+    describe("/POST users", () => {
+        it("the get users endpoint should exist", (done) => {
+            chai.request(usersRoutes).
+                post("/users").
+                set("content-type", "application/json").
+                send({
+                    email: "user@email.com",
+                    name: "user",
+                    password: "1234"
+                }).
+                end((err, res) => {
+                    should.not.exist(err);
+                    res.should.have.status(200);
+                    done();
+                });
+        });
+    });
+});
```

Comment:
eventually these tests should check if the response body is correct

Reply:
Yup! this is just setting up the framework with examples of how to do the test. Next step is to generate properly formatted responses and testing them, as well as setting up a test db (maybe keyspace will do)